packages/suite/src/views/wallet/send/Outputs/Address.tsx

```diff
@@ -303,7 +334,15 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
         }
         if (addressError) {
             return (
-                <InputError message={addressError.message} button={getValidationButtonProps()} />
+                <InputError
+                    message={addressError.message}
+                    learnMoreAsTextUrl={
+                        addressError.type === 'contract'
+                            ? HELP_CENTER_EVM_SEND_TO_CONTRACT_URL
+                            : undefined
+                    }
```

Comment:
this is not nice.

instead please improve the `InputError` component and also `button={getValidationButtonProps()}`
can be used somehow like this
```
{...getInputErrorProps()}
```

Reply:
Did this in the fixup commit